maxmin/max3/max3.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+/* (maxmin/max2) a program that calculates the maximum of three numbers.*/
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	fmt.Println("The program finds the maximum of three numbers.")
+	fmt.Println("Enter three numbers:")
+	var a float64
+	var b float64
+	var c float64
+	fmt.Scan(&a, &b, &c)
+	var max1 = math.Max(a, b)
```

Comment:
How many comparisons do you have to make to get the largest of 3 numbers? Is it 3 or 2? If you know `max(a, b)`, then what would be `max(max(a, b), c)`?

Reply:
The same for min3

Reply:
2, if use result of first comparison to with compare with the remaining value, but I forgot about this version.

Reply:
Please rewrite max/min programs using formulas, i.e. without using `math.Max` and `math.Min`.